pkg/backup/main.go

```diff
@@ -21,6 +22,18 @@ var (
 	log     = logrus.WithFields(logrus.Fields{"pkg": "backup"})
 )
 
+type CreateBackupParameters struct {
```

Comment:
ref: https://stackoverflow.com/a/45886055 

I think the original naming should be fine. Or if you want it more precise, you can use `BackupCreateParameter`. 

nit: I feel we can do this pattern for the parameter for DoBackupRestore as well. This is not a priority. We can refactor over time.

Reply:
`BackupCreateParameter` sounds good.